frontend/src/App.less

```diff
@@ -85,12 +107,6 @@ body {
 .modal-body {
     height: ~'calc(100vh - 150px)';
 
-    .uast-editor,
```

Comment:
are you sure we don't need it?

I remember one of the reasons to add it was "This site appears to use a scroll-linked positioning effect. This may not work well with asynchronous panning; see https://developer.mozilla.org/docs/Mozilla/Performance/ScrollLinkedEffects for further details and to join the discussion on related tools and features!".

Reply:
nwm. I see you moved it to another file.